spec/src/main/asciidoc/app-programming-model.adoc

```diff
@@ -699,7 +699,7 @@ MetricRegistry metricRegistry;
 [source, java]
 ----
 @Inject
-@RegistryType(type=MetricRegistry.Type.APPLICATION)
+@RegistryType(scope=MetricRegistry.APPLICATION_SCOPE)
```

Comment:
This line is fine, but the preceding discussion (line 677, which GitHub won't let me flag) still refers to `@RegistryType` as a qualifier and it lists only the three built-in scopes; it would be good to mention briefly user-defined ones there as well.

Reply:
Ah yes, that would be important 😁